Dockerfile

```
@@ -7,7 +7,8 @@ RUN npm ci
 
 COPY . .
 
-RUN npm run build -- --output-path=public/app
-RUN npm run build-storybook
+ENV API_ENDPOINT http://cse-dev.cass.oregonstate.edu
```

This needs to be 
```suggestion
ENV API_ENDPOINT https://cse-dev.cass.oregonstate.edu
```
to connect to the currently-hosted version of the API.